### ISSUE TITLE
snippets-extra: reflection, multi-catch statement not supported in CATG with java 1.6

### ISSUE THREAD

**Issue by micskeiz** [SETTE-Testing/sette-snippets#4]
Some of the reflection snippets does not compile for CATG, which only supports Java 1.6

```
multi-catch statement is not supported in -source 1.6
...
   } catch (ClassNotFoundException | NullPointerException | IllegalArgumentException ex) {
```

Either modify the snippets or exclude them from CATG.

**Comment by LajosCseppento** [comment]
Updated code base to compile with source level 1.6, however, the code still uses classes introduced in Java 7. Now the project compiles for CATG as well

**Comment by micskeiz** [comment]
Fixed in ffc1234e97c976d885b4df259c622a77781f9d2d
